test/lib/jdk/test/lib/security/SecurityUtils.java

```diff
@@ -48,5 +52,23 @@ public static KeyStore getCacertsKeyStore() throws Exception {
         return KeyStore.getInstance(file, (char[])null);
     }
 
+    /**
+     * Removes the specified protocols from the jdk.tls.disabledAlgorithms
+     * security property.
+     */
+    public static void removeFromDisabledTlsAlgs(String... protocols) {
+        removeFromDisabledAlgs("jdk.tls.disabledAlgorithms",
+                               List.<String>of(protocols));
+    }
+
+    private static void removeFromDisabledAlgs(String prop, List<String> algs) {
```

Comment:
Useful utility method. Maybe it can be made public/ opened up and renamed to something like "removeFromSecurityProperty" perhaps  ? could be done at a later time perhaps.

Reply:
Yes, that's the main reason why I put that code into a separate method, so that we could consider adding more methods that call it or using it more generally in subsequent fixes where we need to re-enable disabled algorithms.